Formula/ford.rb

```diff
@@ -33,11 +33,6 @@ class Ford < Formula
     sha256 "09ed0cde452d015fe77c4845a210eb642f28d245f5bc250d4b97808cb8f49078"
   end
 
-  resource "importlib-metadata" do
```

Comment:
Noting that this getting deleted is a `bump-formula-pr`/`update-python-resource` issue due to `pipgrip` using Python 3.10 and not detecting Python 3.9 dependencies, e.g.
```
importlib-metadata>=4.4;python_version<'3.10'
```